errbot/backends/slack.py

```diff
@@ -200,7 +199,7 @@ def api_call(self, method, data=None, raise_errors=True):
         """
         if data is None:
             data = {}
-        response = json.loads(self.sc.server.api_call(method, **data).decode('utf-8'))
+        response = self.sc.server.api_call(method, **data).json()
```

Comment:
In the referenced PR, [line 135](https://github.com/slackhq/python-slackclient/pull/54/files#diff-599dbee7d2139b2e66b89a120d50a0a1R135) returns `Response.text`, which returns a unicode object. This means it should still be decoded as `utf-8` and fed to `json.loads` (there's no `.json()` available, that's only a function on the original `Response` object).

That means there should be no changes necessary on our side, which in my opinion is how it should be. You don't want to change the public API, as that might break a lot of users of library.